src/ralph/data_importer/tests/test_fields.py

```diff
@@ -0,0 +1,37 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from ralph.accounts.tests.factories import UserFactory
+from ralph.data_importer.fields import ThroughField
+from ralph.data_importer.resources import LicenceResource
+from ralph.data_importer.widgets import LicenceUserWidget
+from ralph.licences.models import Licence, LicenceUser
+from ralph.licences.tests.factories import LicenceFactory
+
+
+class DataImporterFieldsTestCase(TestCase):
+    def setUp(self):  # noqa
+        self.licence = LicenceFactory()
+        self.users = [UserFactory() for i in range(2)]
+        LicenceUser.objects.create(
+            licence=self.licence, user=self.users[0]
+        )
+
+    def test_through_field(self):
+        field = ThroughField(
+            through_class=LicenceUser,
+            through_field_obj='licence',
+            through_field='user',
+            attribute='users',
+            column_name='users',
+            widget=LicenceUserWidget(model=get_user_model())
+        )
+
+        self.assertEqual(self.licence.users.all().count(), 1)
+
+        field.save(
```

Comment:
could you make assertion on number of queries here? https://docs.djangoproject.com/en/dev/topics/testing/tools/#django.test.TransactionTestCase.assertNumQueries